### PR TITLE
Documentation for the Template

### PR DESCRIPTION
**Pull request by danielguinan1**

### CODE REVIEW

**Review by gljrobinson — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -1 +1,35 @@
-# liberty-backstage-demo
+# Open Liberty Backstage Template
+This is a sample template for Backstage/Red Hat Developer Hub that creates and deploys the Liberty Starter app on Kubernetes.  Though this template may work as is, the best practice for templates in Backstage to be designed by Platform Engineers for their specific infrastructure.  This template should be used as a base for designing your own template in Backstage.
```
Comment (erica-banda-03) [inline README.md]: Should this be something like this? - "Though this template may work as is, the best practice is for Platform Engineers to design templates specific to their infrastructure."

File: README.md [hunk 2]
```diff
@@ -1 +1,35 @@
-# liberty-backstage-demo
+# Open Liberty Backstage Template
+This is a sample template for Backstage/Red Hat Developer Hub that creates and deploys the Liberty Starter app on Kubernetes.  Though this template may work as is, the best practice for templates in Backstage to be designed by Platform Engineers for their specific infrastructure.  This template should be used as a base for designing your own template in Backstage.  
+
+## What does it do?
+This template after taking parameters from Backstage/RHDH does the following:
```
Comment (erica-banda-03) [inline README.md]: I think we should put "after taking parameters from Backstage/RHDH" in parenthesis. Let me know what you think. 
This template (after taking parameters from Backstage/RHDH) does the following:

File: README.md [hunk 3]
```diff
@@ -1 +1,35 @@
...
+  - Getting Started with [Backstage](https://backstage.io/docs/getting-started/)
+  - Getting Started with [RedHat Developer Hub](https://developers.redhat.com/rhdh/getting-started)
+- GitHub Application created and connected to your instance. ([Backstage](https://backstage.io/docs/auth/github/provider#create-an-oauth-app-on-github)) or ([RHDH](https://developers.redhat.com/learning/learn:openshift:install-and-configure-red-hat-developer-hub-and-explore-templating-basics/resource/resources:configure-github-access-red-hat-developer-hub))
+- Kubernetes Frontend and Backend Plugins enabled and configured for the deployment location. ([Backstage](https://backstage.io/docs/features/kubernetes/installation)) or ([RHDH](https://access.redhat.com/documentation/en-us/red_hat_developer_hub/1.1/html/administration_guide_for_red_hat_developer_hub/rhdh-installing-dynamic-plugins))
+- ArgoCD installed and set up. [ArgoCD Docs](https://argo-cd.readthedocs.io/en/stable/getting_started/)
+- ArgoCD Plugins added to Backstage 
+  - Frontend: ([Backstage](https://github.com/RoadieHQ/roadie-backstage-plugins/tree/main/plugins/frontend/backstage-plugin-argo-cd)) or (RHDH: )
```
Comment (erica-banda-03) [inline README.md:20]: Do we have the RHDH links for these? - `(RHDH: )`

File: README.md [hunk 4]
```diff
@@ -1 +1,35 @@
...
+  - Scaffolder: ([Backstage](https://www.npmjs.com/package/@roadiehq/scaffolder-backend-argocd))
+  
+- GitHub Actions plugin installed and set up. ([Backstage](https://github.com/backstage/community-plugins/tree/main/workspaces/github-actions/plugins/github-actions))
+- Open Liberty Operator installed on your cluster ([Docs](https://github.com/OpenLiberty/open-liberty-operator/blob/main/doc/user-guide-v1.adoc))
+
+## Usage
+Once all the prerequisites are set up you can use this template with the following steps:
+1. Click "Create" in your Backstage instance.
+2. Click "Register Existing Component" on the top right.
+3. Add the following link into the URL: `https://github.com/OpenLiberty/liberty-backstage-demo/blob/main/liberty-template/template.yaml` and click Analyze.
```
Comment (erica-banda-03) [inline README.md]: For consistency, I think `Analyze` should be `"Analyze"`.

**Review by erica-banda-03 — COMMENTED** [review]
This looks great!! Thank you so much for writing this up!
I commented on a few minimal changes I recommend. Let me know what you think.

File: README.md (hunk 3 above)
Comment (danielguinan1) [reply README.md:20]: Not as of 1.2 docs.  Will further investigate.

**Pull request merged**